core/jobs/types/firebase_adapters_test.py

```
@@ -0,0 +1,84 @@
+
+from unittest import mock
+
+from core.constants import constants
+from core.jobs.types import firebase_adapters
+from core.tests import test_utils
+
+
+class RecordTests(test_utils.GenericTestBase):
+
+    def test_strong_record_init(self) -> None:
+        strong_record = firebase_adapters.StrongRecord('aid', 'a@a.com', False)
```

Ruff (0.15.0)

[error] Boolean positional value in function call

(FBT003)